src/azul/project/hca/metadata_generator.py

```diff
@@ -29,7 +29,7 @@ def __init__(self, order=None, ignore=None, format_filter=None):
             "^analysis_process.*",
             "^process.*",
             "^bundle_.*",
-            "^\\*\\.provenance\\.update_date"
```

Comment:
This deletion seems like an unrelated change. What does it do? And shouldn't it be committed separately?

Reply:
Hey @GPelayo, I could not remember the explanation you gave me for removing this field. Could you elaborate to @hannes-ucsc on the reason for the removal? Thank you in advance.

Reply:
@hannes-ucsc  The `*.provenance.update_date` column was added in the full metadata TSV because originally  I thought that field was synonymous with the `file_version`. Since we know now that the `*.provenance.update_date` is not the `file_version` and, we are extracting the file_version from `bundle.json`, I advised @achave11 to remove it.

Reply:
Got it. Thank you, @GPelayo